readme.md

```diff
@@ -17,6 +17,10 @@ You can try a live demo [here](https://codesandbox.io/s/dazzling-moon-itop4).
 ```bash
 npm install zustand
 ```
+#### or
+```bash
+yarn add zustand
+```
```

Comment:
we prefer to slim readme as much as possible. do you have an idea to make it compact? how about this?
```bash
npm install zustand # or yarn add zustand
```

Reply:
Ok, I will do that.
but still, the readme is too heavy.
we should have a better doc